app.rb

```diff
@@ -0,0 +1,88 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_reader :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    puts 'List of all people:'
+    people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
+    end
+  end
+
+  def create_person(name, age, role, _classroom = nil)
+    if role == 'student'
```

Comment:
- [ ] You did a great job setting up the console UI. :+1: Some of the functionalities are not currently working. When trying to create a student, I encountered the following error. Kindly fix it. 🤝️

![Screenshot from 2023-07-01 06-17-40](https://github.com/Christelle-12/school_library/assets/17243325/a4ee8985-10c4-49ac-a46b-94ec9b96020a)